go.mod

```diff
@@ -3,8 +3,11 @@ module github.com/intel/telemetry-aware-scheduling
 go 1.13
 
 require (
-		k8s.io/api v0.20.2
-    	k8s.io/apimachinery v0.20.2
-    	k8s.io/client-go v0.20.2
-    	k8s.io/metrics v0.20.2
+	github.com/smartystreets/goconvey v1.6.4
```

Comment:
In PR #53 we're removing the top level go mod. This may impact a small part of this structure, but it shouldn't cause any real inconvenience. It would be good to get that changed merged first and this rebased.

Reply:
I need to rebase, #53 ought to be merged first